templates/hesabe_app/pay.html

```diff
@@ -15,14 +15,14 @@ <h5>Hesabe Payment gateway</h5>
 			<input type="text" hidden name="variable3" value = {{variable1}}>
 			<input type="text" hidden name="variable4" value = {{variable1}}>
 			<input type="text" hidden  name="variable5" value = {{variable1}}>
```

Comment:
all the values in the form have only `variable1` as the value. They should be concurrent with their name.

Reply:
noted.

Reply:
Also, these fields should be visible cause merchants may use these parameters for reference